tests/functional-tests/README.md

```diff
@@ -16,62 +13,60 @@ This project contains an automation framework to exercise the IBM Spectrum Scale
 
 Install Python (3.7.4 or higher) and below mentioned pip modules:
 
-  ``` 
-        python3.7 -m pip install kubernetes
-        python3.7 -m pip install pytest
-        python3.7 -m pip install pytest-html
-        python3.7 -m pip install jsmin
-  ```
-       
+``` 
+python3.7 -m pip install kubernetes
+python3.7 -m pip install pytest
+python3.7 -m pip install pytest-html
+python3.7 -m pip install jsmin
+```
 
 ### How to run IBM Spectrum Scale CSI test automation
-- Clone the source code from [git](https://github.com/IBM/ibm-spectrum-scale-csi/) repository.
-
-- Configure parameters in [tests/functional-tests/config.json](https://github.com/IBM/ibm-spectrum-scale-csi/blob/dev/tests/functional-tests/config.json) file. 
-
-- Set node labels for "attacherNodeSelector","provisionerNodeSelector","pluginNodeSelector" in config.json file
-
-
-- cluster-config parameter is mandatory to pass the SpectrumScale & CSI Operator/Driver images configuration as input.
 
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json
-                   
-- If kubeconfig file is at not at ~/.kube/config location, pass the correct location with --kubeconfig (optional)               
-                
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json --kubeconfig <kubeconfig_file_path>
+- Configure parameters in [config.json](./tests/functional-test/config.json) file.
+
+- Define kubernetes/Openshift Spectrum Scale node labels for `attacherNodeSelector`,`provisionerNodeSelector`,`pluginNodeSelector` in config.json file
+
+- `--clusterconfig` parameter is mandatory to pass IBM Spectrum Scale API Credentials & CSI Operator/Driver image specific configuration as input.
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json
```

Comment:
FYI, These spaces are not needed ahead of the content, it will render correctly.  But I see why it was added, because of the "For Example" above it.